go/libkb/features.go

```diff
@@ -156,6 +156,14 @@ func (s *FeatureFlagSet) EnabledWithError(m MetaContext, f Feature) (on bool, er
 		"features": S{Val: string(f)},
 	}
 	err = m.G().API.GetDecode(m, arg, &raw)
+	switch err.(type) {
```

Comment:
can leave this in too as a sanity check but shouldn't we check before making the API request so we don't mash the endpoint each time?